### ISSUE TITLE
419 error when attempting to change roles after logging in with "remember me" selected

### ISSUE THREAD

**Issue by jobara** [accessibility-in-action/platform#1512]
**Prerequisites**

- [x] I've [searched](https://github.com/accessibility-exchange/platform/issues) for existing issues

**Describe the bug**

A 419 error is thrown when attempting to change a user's roles after logging in with "remember me" selected. I've experienced this with Individual users, but haven't tested Organizations yet.

**To Reproduce**
1.  Create a new Individual User
2. Verify the account
3. Sign Out
4. Sing back in as the same user and check "remember me"
5. From the dashboard open the "Edit Roles" link
6. Change the roles
7. Click "Update"
8. Notice the 419 error
9. Sign out and repeat the steps from 4 - 8 but without checking "remember me". Notice that it works.

**Expected behavior**

Should be able to change roles without errors.

**Additional context**

Maybe related to #1165

**Comment by michelled** [comment]
Logan has suggested that if it's faster to remove the 'remember me' feature we should do that instead.

**Comment by jobara** [comment]
Looked into this today. It seems there is something wrong with the session and/or CSRF token when the `remember me` option is selected. The CSRF token is updated each time it is accessed. This seems to be both on page loads/reloads and also possibly when performing the check to verify the token. It's unclear why this is happening though, it should persist for the life of the session. It's possible the session is dying, and I have noticed while doing `dd` statements that the session is removed from the database; however it's restored without the user being logged off. Albeit with a different token.

**Comment by jobara** [comment]
Walking through the execution I found that login process finished correctly; however, the session was being remove and replaced once the redirect was executed. In fact on each page load it would happen. I dug in all the way down into Laravel's connection to the MySQL database and found that the data was being corrupted before being sent to the database. It turns out that there is hook that executes on the query before the statement is generated. 

A package called [spatie/laravel-sql-commenter](https://github.com/spatie/laravel-sql-commenter) was executing and causing the corruption. It appears that this package was added for debugging purposes at https://github.com/accessibility-exchange/platform/pull/566/commits/5deacd4955b05ca02c869615e0bda4b44adcab89 as part of the work in https://github.com/accessibility-exchange/platform/pull/566 and merged at https://github.com/accessibility-exchange/platform/commit/c9e92df907a3d08a34033beef02f4f4c20bc9990

I couldn't see how to file an issue with them, so for now I'm just going to remove the package and if we need it later we'll have to investigate if it can be used without causing this bug.
